server/views/applications/show.njk

```diff
@@ -40,10 +41,19 @@ printButton %}
 
       {{ applicantDetails(application) }}
 
-      {% if application.document.sections | length %}
-        {{ documentSummaryList(application) }}
-      {% endif %}
+      <div class="govuk-grid-column-one-quarter side-nav">
 
+        {% if application.document.sideNavItems | length %}
+          {{sideNav(application.document.sideNavItems)}}
```

Comment:
Instead of saving the nav items to the application document, could we generate them using a helper when we render the page?

```
{{ sideNav(generateNavLinks()) }}
```

With `generateNavLinks` making use of `getSections`.

Reply:
yeah I was thinking similar thing, there could be a way to do this without adding a UI-specific field to the `document`, it would just involve moving around where we do the javascript?... I have a WIP branch that I can share

Reply:
I've got a very rough branch here, we could chat as a team about the options 
https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/compare/cas2-68-side-nav-hw?expand=1